classes/log/file.php

```diff
@@ -72,7 +72,7 @@ public static function write($level, $msg, $method = null)
 	
 						//
 						if($level === 'Error'){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.ls.aws.cybird.ne.jp:9000/coding_rules#rule_key=php%3AS134)